boring/src/ssl/mod.rs

```diff
@@ -3109,6 +3166,78 @@ impl SslRef {
         str::from_utf8(version.to_bytes()).unwrap()
     }
 
+    /// Sets the minimum supported protocol version.
+    ///
+    /// A value of `None` will enable protocol versions down the the lowest version supported by
+    /// OpenSSL.
```

Comment:
:see_no_evil: 

We might want to stop copying docs from OpenSSL...

Reply:
Fixed, after investigation it seems that BoringSSL won't ever return `0` for `get_{min|max|_proto_version`, so those APIs returning Options doesn't make much sense. We should fix that in the next breaking change